spec/features/workflow/create_document_spec.rb

```diff
@@ -14,22 +14,26 @@ def given_i_am_on_the_home_page
   end
 
   def when_i_click_to_create_a_document
-    @document_type = build :document_type
     click_on "Create new document"
   end
 
   def and_i_select_a_supertype
-    choose I18n.t("supertypes.news.label")
+    choose I18n.t("document_type_selections.news.label")
     click_on "Continue"
   end
 
   def and_i_select_a_document_type
-    choose @document_type.label
+    choose I18n.t("document_type_selections.news_story.label")
     click_on "Continue"
   end
 
   def and_i_fill_in_the_contents
     stub_any_publishing_api_put_content
+    base_path = Edition.last.document_type.path_prefix + "/a-title"
+    content_id = Document.last.content_id
+    stub_publishing_api_has_lookups(base_path => content_id)
+    stub_publishing_api_has_links("content_id" => content_id)
```

Comment:
Nice work fixing these, such a bummer they creep through the stack. I think you might be able to make these a bit simpler.

From looking at the code I think `stub_publishing_api_has_lookups({})` might suffice rather than needing to work out the base path. Which could mean has_links ends up as: `stub_publishing_api_has_links(content_id: Document.last.content_id)`